bigbluebutton-tests/playwright/chat/chat.spec.js

```diff
@@ -17,17 +19,30 @@ test.describe.parallel('Chat test suite', () => {
     await clear.test();
   });
 
-  test('Copy chat', async ({ browser, context, page }) => {
-    test.fixme(true, 'Only works in headed mode');
+  test('Copy chat', async ({ browser, context, page }, testInfo) => {
+    test.fixme(testInfo.config.projects[0].use.headless, 'Only works in headed mode');
```

Comment:
shouldn't we use `testInfo.project.use.headless` here? it looks like it will get this data dynamically - from the project that is currently running. (I searched about this but I'm not sure about it, I haven't tested it either)

in a near future, when we have more projects set up, maybe we will have different configurations